storage/persistent_storage/rosa-persistent-storage-aws-ebs.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: ASSEMBLY
+[id="rosa-persistent-storage-aws-ebs"]
+= Persistent storage using AWS Elastic Block Store (EBS)
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-persistent-storage-aws-ebs
+
+toc::[]
+
+{product-title} (ROSA) clusters are pre-built with two storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
+
+Following are the two pre-built storage classes:
+[options="header"]
+
+|===
+
+| Name | Provisioner
+
+| gp2 | kubernetes.io/aws-ebs
+
+| gp2-csi | ebs.csi.aws.com
+
+|===
+
+[NOTE]
+====
+The gp2 storage class is set as default; however, you can select either one as the default storage class.
+====
```

Comment:
Consider just adding `(default)` next to the `gp2` line in the table, or possibly just making this a regular sentence in one of the paragraphs.

Reply:
Moved the text into a regular sentence in a paragraph below the table.